app/queries/linked_items_query.rb

```diff
@@ -0,0 +1,84 @@
+class LinkedItemsQuery
+  attr_reader :content_item
+
+  def initialize(content_item)
+    @content_item = content_item
+  end
+
+  def call
+    items = linked_items
+    items["available_translations"] = available_translations if available_translations.any?
+    if content_item.format == "working_group"
+      items["policies"] = content_items_by_id.values.select { |content_item| content_item.format == "policy" }
+    end
+    items
+  end
+
+private
+
+  # Return the extant and renderable linked_items and any "passthrough" items
+  # which aren't really content_items, but are instead hardcoded hashes.
+  def linked_items
+    content_item.links.each_with_object({}) do |(link_type, raw_content_ids), result|
+      passthrough_hashes, raw_content_ids = raw_content_ids.partition { |link| link.is_a?(Hash) }
+      passthrough_content_items = passthrough_hashes.map do |attributes|
+        ContentItem.new(attributes)
+      end
+
+      real_content_items = raw_content_ids.map { |id| content_items_by_id[id] }.compact
+      result[link_type] = real_content_items + passthrough_content_items
+    end
+  end
+
+  # Return the renderable items based on `content_ids`, in the most appropriate
+  # locale
+  def content_items_by_id
+    defined?(@_content_items) ? @_content_items : @_content_items = begin
```

Comment:
It avoids accidental recomputation if the return value is falsey, which is a problem with:
```ruby
@_content_items ||= ...
```